Revamp the Ballerina VS Code extension docs (#213)
wso2/docs-ballerina-vscode-extension: en/docs/design-the-project/design-using-the-architecture-view.md

> # Design using the architecture view

Reviewer: If `Architecture View` is the UI component why using lower case letters? In the following paragraph it has upper case letters which his inconsistent.

Author: Fixed.

> The Architecture View includes a range of editing capabilities that makes it possible to design projects via the diagram. The sections below explore these capabilities by executing the **Architecture View** command on an empty workspace.

Reviewer: ```suggestion
The Architecture View includes various editing capabilities that make it possible to design projects via the diagram. The sections below explore these capabilities by executing the **Architecture View** command on an empty workspace.
```

## Add components

> Use the **Add Component** option of the Architecture View to add the first component of your project as shown below. 

Reviewer: ```suggestion
Use the **Add Component** option of the Architecture View to add the first component of your project, as shown below. 
```
